cfgov/jinja2/v1/_includes/templates/forms/submit-a-request-form.html

```diff
@@ -1,218 +1,276 @@
+                'size':         '1-2',
+                'required':     true,
+                'disabled':     false
+            }
+        }) %}
+        {% set value = global_dict.input %}
+        {% include 'atoms/input.html' with context %}
 
-            {{ form_field.date_picker('Document dates:') }}
+        {% do global_dict.update({
+            'date_picker': {
+                'legend':       'Document dates:'
+            }
+        }) %}
+        {% set value = global_dict.date_picker %}
+        {% include 'atoms/date-picker.html' with context %}
 
-            <div class="form-l_col form-l_col-1">
-                <label for="form_requested-info">
-                    <span class="form-label-header">Requested information</span>
-                    <span class="micro-copy micro-copy__large">
-                        Describe the records you seek from the CFPB.
-                        <a href="../foia-faqs" target="_blank">
-                            Review our FAQs for additional guidance.
-                        </a>
-                    </span>
-                </label>
-                <textarea class="input__long"
-                          id="form_requested-info"
-                          ></textarea>
-            </div>
 
-            {% do global_dict.update({
-                'select': {
-                    'label':        'Select a request category',
-                    'placeholder':  'Select FOIA request category',
-                    'fields':       ['Commercial', 'Media', 'Others'],
-                    'size':         '1',
-                    'required':     false,
-                    'disabled':     false
-                }
-            }) %}
-            {% set value = global_dict.select %}
-            {% set prototype = true %}
-            {% include 'atoms/select.html' with context %}
+        {% do global_dict.update({
+            'textarea': {
+                'label':        'Requested information',
+                'helperText':   'Describe the records you seek from the CFPB.
+                                 <a href="../foia-faqs">Review our FAQs
+                                 for additional guidance.</a>',
+                required:       true
+            }
+        }) %}
+        {% set value = global_dict.textarea %}
+        {% include 'atoms/textarea.html' with context %}
+
+        {% do global_dict.update({
+            'select': {
+                'label':        'Select a request category',
+                'placeholder':  'Select FOIA request category',
+                'fields':       ['Commercial', 'Media', 'Others'],
+                'size':         '1',
+                'required':     false,
+                'disabled':     false
+            }
+        }) %}
+        {% set value = global_dict.select %}
+        {% include 'atoms/select.html' with context %}
 
-            <div class="js-additional-field">
+        <div class="js-additional-field">
+        </div>
```

Comment:
It's just an extra block, I have additional_field class on fieldset right under this. Deleted in 1389c24